Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM node:18
+FROM node:18 as Builder
+
+WORKDIR /action
+
+RUN npm i -g @vercel/ncc
```

Comment:
Should this line be removed now that it's in `yarn build`?

Reply:
Oh I thought that I removed it. Doing it now